### PR TITLE
Fixed encoding problem

### PR DESCRIPTION
**Pull request by BrianPetkovsek**
yes... all that was needed was two spaces at the top.

#106

### CODE REVIEW

**Comment by k4yt3x** [comment]
Does this format comply with Python conventions and standards?
I kinda want clarity and standardization over compatibility.

File: bin/video2x_setup.py [hunk 1]
```diff
@@ -87,6 +89,7 @@ def run(self):
     def _install_python_requirements(self):
         """ Read requirements.txt and return its content
         """
+        pip_uninstall('requirements.txt')
```
Comment (sat3ll) [inline bin/video2x_setup.py]: why uninstall what might be already available?

Comment (BrianPetkovsek) [reply bin/video2x_setup.py]: it was in there for my own purposes 
idk why i committed it in (removed now)

**Comment by BrianPetkovsek** [comment]
> Does this format comply with Python conventions and standards?
> I kinda want clarity and standardization over compatibility.

@k4yt3x from what it looks like it does comply 
what the program does is replace the f-string which was implemented in 3.6 to the format function

```python
# -*- coding: future_fstrings -*-
thing = 'world'
print(f'hello {thing}')
```
would be transformed to:
```python
# -*- coding: future_fstrings -*-
thing = 'world'
print('hello {}'.format((thing)))
```

**Comment by k4yt3x** [comment]
I'm merging the fix first as the bug is influencing a lot of people, but I'm treating this as a temporary mitigation. The conversations should carry on.

**Pull request merged**